### ISSUE TITLE
Angular Material tooltip bug

### ISSUE THREAD

**Issue by allanatorr** [angular/components#24681]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### The previous version in which this bug was not present was

11.2.13

### Description

Hello,

I found a bug in the newest version of angular material. If you hover fastly over a component (e.g. a button) which has a tooltip with a delay in it, it's nevertheless flashing and showing up the tooltip instantly for a moment and then it disappears again.

Here is my HTML:

<button [matTooltip]="'Test'" [matTooltipShowDelay]='1000'>Test </button> 

### Reproduction

Steps to reproduce:
1. Create button with tooltip and a delay 
2. Hover fastly over the button

### Expected Behavior

Tooltip shouldn't appear, only after one second

### Actual Behavior

Tooltip appears, altough one second didn't passed

### Environment

- Angular: Current
- CDK/Material: Current
- Browser(s): Chrome/Firefox
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by volvachev** [comment]
I think it's similar to this one. (#24614).

**Comment by crisbeto** [comment]
Closing as a duplicate of #24614.
